plugins/slicer/MONAILabel/MONAILabel.py

```diff
@@ -1828,15 +1904,31 @@ def onClearScribblesSegmentNodes(self):
         for i in range(num_segments):
             segmentId = segmentation.GetSegmentation().GetNthSegmentID(i)
             if "scribbles" in segmentId:
-                logging.info("clearning {}".format(segmentId))
+                logging.info("clearing {}".format(segmentId))
                 labelMapRep = slicer.vtkOrientedImageData()
                 segmentation.GetBinaryLabelmapRepresentation(segmentId, labelMapRep)
                 vtkSegmentationCore.vtkOrientedImageDataResample.FillImage(labelMapRep, 0, labelMapRep.GetExtent())
                 slicer.vtkSlicerSegmentationsModuleLogic.SetBinaryLabelmapToSegment(
                     labelMapRep, segmentation, segmentId, slicer.vtkSlicerSegmentationsModuleLogic.MODE_REPLACE
                 )
 
+        # refresh segmentation view to clear scribbles segmentations
+        # help from: https://discourse.slicer.org/t/refresh-volume-rendering/11847/6
+        segmentation.SetDisplayVisibility(False)
+        segmentation.SetDisplayVisibility(True)
+
     def onClearScribbles(self):
+        # for clearing scribbles and resetting tools to default
+        # remove "scribbles" segments from label
+        self.onClearScribblesSegmentNodes()
+
+        self.ui.paintScribblesButton.setChecked(True)
+        self.ui.eraseScribblesButton.setChecked(False)
+
+        self.ui.scribblesSelector.setCurrentIndex(0)
```

Comment:
how about ROI.. will it be good idea to use a specific named ROI for scribbles.. similar to foreground/background scribbles.. instead of any ROI node..

Reply:
I need to think a bit about this one. 
There could be many ways in which we utilise ROIs. 

Currently, they are used to limit the "working area" for scribbles, as such anything outside is ignored (e.g. by adding background scribbles outside or any other similar methods on server) - so the current flow does not require them to be specific for background/foreground etc.

Reply:
We certainly need to reset ROIs for each new label selection in scribbles - I will look into this. 
I think we can also add an ROI button within scribbles panel to help a user quickly add ROI.